nodejs/week3/warmup/index.js

```diff
@@ -0,0 +1,130 @@
+const express = require("express");
+const app = express();
+
```

Comment:
I think you are missing here

```suggestion
app.use(express.urlencoded({extended: true}))
app.use(express.json());
```